### ISSUE TITLE
Possibility to add feed?

### ISSUE THREAD

**Issue by visualcookie** [shobhitsharma/embedo#6]
Hi,

so far so good. I like the codebase quite a lot and like how easy it actually is, compared to go to each social network and generate the thing and copy their *cough* shitty code.

Is it possible or planned, that it will also generate whole feeds? For example, when loading in only a pages URL or users URL.

**Comment by shobhitsharma** [comment]
@visualcookie With feed do you mean RSS or timeline embeding? Between, it already embeds twitter timeline, for [facebook](https://developers.facebook.com/docs/plugins/page-plugin) I'll add support for page embeds later this week.

**Comment by visualcookie** [comment]
Yeah, I mean whole timelines. For instance a grid with pictures of your Instagram feed etc
